docs/release/support_matrix_v2_0.md

```diff
@@ -1,73 +1,273 @@
-It is strongly recommended to use exactly those libfabric versions.
+
+DAOS Version 2.0 requires libfabric Version 1.14. The RPM distribution
+of DAOS includes libfabric RPM packages with the correct version.
+It is strongly recommended to use exactly the provided libfabric version
+on all DAOS servers and all DAOS clients.
 
 Not all libfabric providers are supported.
 DAOS Version 2.0 has been validated mainly with the `verbs` provider
 for InfiniBand fabrics, and the `sockets` provider for other fabrics.
-Note that the `psm2` provider for Omni-Path fabrics has known issues
+Future DAOS releases may add support for additional libfabric providers.
+
+Note: The `psm2` provider for Omni-Path fabrics has known issues
 when used in a DAOS context, and is not supported for production
-environments.
-
-The following table lists the
-[MLNX\_OFED](https://www.mellanox.com/products/infiniband-drivers/linux/mlnx_ofed)
-versions that have been validated with the libfabric verbs provider
-for the different DAOS release tags.
-Other MLNX\_OFED versions may work, but have not been tested.
-
-| **Tag** | MLNX\_OFED  |
-|---------|-------------|
-| v1.0.1  | 5.0-1       |
-| v1.2    | 5.1-x       |
-| v2.0    | 5.3-1/5.4-1 |
-| v2.2    | tbd         |
-
-Information for future releases is indicative only and may change.
+environments. Please refer to the
+[Cornelis Networks presentation](https://daosio.atlassian.net/wiki/download/attachments/11015454821/12_Update_on_Omni-Path_Support_for_DAOS_DUG21_19Nov2021.pdf)
+at [DUG21](https://daosio.atlassian.net/wiki/spaces/DC/pages/11015454821/DUG21)
+for an outlook on future Omni-Path Express support for DAOS.
+
+On InfiniBand fabrics with the `verbs` provider, DAOS requires that the
+[Mellanox OFED (MLNX\_OFED)](https://www.mellanox.com/products/infiniband-drivers/linux/mlnx_ofed)
+software stack is installed on the DAOS servers and DAOS clients.
+DAOS Version 2.0 has been validated with MLNX\_OFED Version 5.3-1 and
+Version 5.4-1.
+It is strongly recommended that all DAOS servers and all DAOS clients
+run the same version of MLNX\_OFED, and that the InfiniBand adapters are
+updated to the firmware levels that are included in that MLNX\_OFED
+distribution.
+
+
+## DAOS Scaling
+
+DAOS is a scale-out storage solution that is designed for extreme scale.
+This section summarizes the DAOS scaling targets, some DAOS architectural limits,
+and the current testing limits of DAOS Version 2.0.
+
+Note: Scaling characteristics depend on the properties of the high-performance
+interconnect, and the libfaric provider that is used. The DAOS scaling targets
+below assume a non-blocking, RDMA-capable fabric. Most scaling tests so far
+have been performed on InfiniBand fabrics with the libfabric `verbs`provider.
+
+DAOS scaling targets (order of magnitude):
+* DAOS client nodes in a DAOS system:   10<sup>6</sup> (hundreds of thousands)
+* DAOS servers in a DAOS system:        10<sup>3</sup> (thousands)
```

Comment:
If I were a sales person reading this, I would assume I'm ok to sell a DAOS system with 9,000 servers and expect it to be expected to work on DAOS 2.0, we just don't know otherwise. Is that what we want to convey? If we have known limits in 2.0, should we convey those?

Reply:
I have to agree.  Unless we have done some testing that I am not aware of, this sounds like it's inviting scale that is orders of magnitude beyond what we have been able to test.  Are we committing to something that we could end up not actually being able to support.

Reply:
It is pretty clear here that this is scaling targets and what is actually tested (to date) is listed below.